sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2100,8 +2100,7 @@ class DataSourceV2SQLSuite
     val e1 = intercept[AnalysisException] {
       sql("DESCRIBE FUNCTION default.ns1.ns2.fun")
     }
-    assert(e1.message.contains(
-      "The namespace in session catalog must have exactly one name part: default.ns1.ns2.fun"))
+    assert(e1.message.contains("Unsupported function name 'default.ns1.ns2.fun'"))
```

Comment:
nit: Is it okay to remove the information about why a given name is unsupported?

Reply:
So the reason isn't that the namespace in session catalog must have exactly one name part anymore, so I think it's an OK change.

Reply:
Do you mean now we have the allowed case? Do we have a test case when it works if so?